src/app/plugins/dog.plugin.ts

```diff
@@ -89,21 +94,19 @@ export class DogPlugin extends Plugin {
       });
   }
 
-  async _makeBreedEmbed(): Promise<MessageEmbed> {
+  private async _makeBreedEmbed(): Promise<MessageEmbed> {
     if (this._breedEmbed) {
       return this._breedEmbed;
     }
 
-    this._breedEmbed = new MessageEmbed();
-
     this._breedEmbed = await this.container.messageService.generateEmbedList(this._breeds, {
```

Comment:
Not sure why `this.container.messageService.generateEmbedList` is async, you mind changing that real quick